packages/react-diagrams-routing/src/link/RightAngleLinkWidget.tsx

```diff
@@ -120,8 +120,6 @@ export class RightAngleLinkWidget extends React.Component<RightAngleLinkProps, R
 				points[index - 2].setPosition(_points[index - 2]);
 				points[index + 1].setPosition(_points[index + 1]);
 				points[index - 1].remove();
-				points[index - 1].remove();
```

Comment:
This might actually be expected behaviour, have you tested this as well? Or was it something that you ran into (I was not the original author of this specific code)